src/Workspaces/Core/Portable/Storage/SQLite/SQLitePersistentStorage_BulkPopulateIds.cs

```diff
@@ -130,14 +131,20 @@ bool AddStrings(HashSet<string> stringsToAdd)
                     {
                         try
                         {
-                            connection.RunInTransaction(() =>
+                            connection.RunInTransaction((ValueTuple<HashSet<string>, PooledObject<Dictionary<int, string>>, SqlConnection> tuple) =>
```

Comment:
Can't seem to figure out the syntax to use valuetuple here.

Reply:
Should be:  `RunInTransaction(((stringsToAdd, idToString, connection) tuple) => ```

Reply:
Doesn't seem to work, complains about type inference.

Reply:
Fixed!